### ISSUE TITLE
on_delete-Error in models.LogsClipboard

### ISSUE THREAD

**Issue by thaeny-dev** [fireantology/django-logtailer#10]
Using django-logtailer in Django 2.2 you will get an error: 
``` 
File "/opt/fmh/cookiecutter-django/fmh-editors/venv/lib/python3.7/site-packages/logtailer/models.py", line 29, in <module>
    class LogsClipboard(models.Model):
File "/opt/fmh/cookiecutter-django/fmh-editors/venv/lib/python3.7/site-packages/logtailer/models.py", line 33, in LogsClipboard
    log_file = models.ForeignKey(LogFile, verbose_name=_('log file'))
TypeError: __init__() missing 1 required positional argument: 'on_delete'
``` 
during django migrate or starting the development-server.

The fix is to add an 'on_delete=models.CASCADE' to the model:
``` 
class LogsClipboard(models.Model):
    #...
    log_file = models.ForeignKey(LogFile, verbose_name=_('log file'), on_delete=models.CASCADE)
```

**Comment by pyarun** [comment]
@fireantology 
I see that this is fixed in a commit: https://github.com/fireantology/django-logtailer/commit/14f2da106937f03067e6e5ae95bfa34648968e09.

Can you make a new version release!

**Comment by fireantology** [comment]
Yep, need to get the time to review the upgrade for Django 2. Planning to do it this month.

**Comment by fireantology** [comment]
Fixed in latest release 1.1
